README.md

```diff
@@ -41,6 +41,16 @@ must be applied via [patches](patches). See [below](#patches) for more info on p
 
 ## Updating an External Component to a Newer Version
 
+When updating a component that is used in both Arcade and another target repository, follow these steps:
+
+1. **Add the updated component**: Include the component with the updated version as a new submodule in source-build-externals.
```

Comment:
Should call out that this would be separate from any previously existing version.

Reply:
Added a callout [here](https://github.com/dotnet/source-build-externals/pull/280/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R84) and specified steps to make it apparent that these should be two separate submodules.